python/golem_task_api/client.py

```diff
@@ -134,43 +144,55 @@ def __init__(
 class ProviderAppClient:
     DEFAULT_PORT: ClassVar[int] = 50006
 
-    @classmethod
-    def _start_service(
-            cls,
+    def __init__(
+            self,
             service: TaskApiService,
-    ) -> ProviderAppStub:
-        host, port = service.start(
-            f'provider {cls.DEFAULT_PORT}',
-            cls.DEFAULT_PORT,
-        )
-        return ProviderAppStub(
+            port: int = DEFAULT_PORT,
+    ) -> None:
+        self._service = service
+        host, port = service.start(f'provider {port}', port)
+        self._golem_app = ProviderAppStub(
             Channel(host, port, loop=asyncio.get_event_loop()),
         )
+        self._kill_switch = Wrapper()
```

Comment:
Don't we need this kind of switch for `RequestorAppClient` as well?

Reply:
yes, will make an issue if i cant fix that this sprint :)

Reply:
CLAY-410